### PR TITLE
Install and use the glob package

### PR DESCRIPTION
**Pull request by aabounegm**
To improve support for shells that do not expand globs automatically
such as PowerShell and CMD on Windows

### CODE REVIEW

**Comment by aabounegm** [comment]
Also, I see that the function `parseModuleFile` (in `src/Language/Rzk/Syntax.hs`) is not used anywhere. Should it be remove or should I add glob support to it as well?

**Comment by aabounegm** [comment]
Did I mention I hate Nix?
No but really, would it make sense to run `cabal2nix` in the CI? I just cannot install it on Windows.

File: rzk/src/Rzk/Main.hs [hunk 1]
```diff
@@ -58,14 +59,16 @@ parseRzkFilesOrStdin = \case
...
-  paths -> forM paths $ \path -> do
-    putStrLn ("Loading file " <> path)
-    result <- Rzk.parseModule <$> readFile path
-    case result of
-      Left err -> do
-        putStrLn ("An error occurred when parsing file " <> path)
-        error err
-      Right rzkModule -> return (path, rzkModule)
+  paths -> do
+    expandedPaths <- foldMap glob paths
```
Comment (fizruk) [inline rzk/src/Rzk/Main.hs:63]: For each path in `paths`, you should throw an error, if it corresponds to an empty list! Right now it silently says "Everything is ok!" if the glob does not refer to any files (or if there is a typo in a filename).

Comment (fizruk) [reply rzk/src/Rzk/Main.hs:63]: I will fix that.

**Pull request merged**